web_scraper/chicago_scraper.py

```diff
@@ -0,0 +1,155 @@
+import pandas as pd
+import numpy as np
+import time
+import platform
+from dotenv import load_dotenv
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions
+import os
+from supabase import create_client, Client
+from datetime import datetime
+import pytz
+
+load_dotenv()
+
+url: str = os.environ.get("SUPABASE_URL")
+key: str = os.environ.get("SUPABASE_SR")
+
+supabase: Client = create_client(url, key)
+
+current_os = platform.system()
+
+if current_os == "Windows":
+    service = Service(executable_path="drivers/chromedriver.exe")  
+elif current_os == "Darwin": 
+    service = Service(executable_path="drivers/chromedriver")  
+else:
+    raise Exception("Unsupported operating system")
+
+options = webdriver.ChromeOptions()
+options.add_argument("--headless=new")
+
+driver = webdriver.Chrome(options=options, service=service)
+url = "https://webapps1.chicago.gov/licensedcontractors/all"
+
+driver.get(url)
+table = driver.find_element(By.ID, 'paginatedTable')
+titles = table.find_elements(By.TAG_NAME, 'th')
+table_titles = [title.text for title in titles]
+
+table_content = []
+wait = WebDriverWait(driver, 5)
+current = 1
+max = 1
+
+table_rows = wait.until(expected_conditions.presence_of_all_elements_located((By.CSS_SELECTOR,'table#paginatedTable tbody tr')))
+
+for row in table_rows:
+    row_data = [item.text for item in row.find_elements(By.TAG_NAME, 'td')] 
+    table_content.append(row_data)
+
+while current < max: 
```

Comment:
are we only doing one page? Have you tested this with multiple pages? We should make this dynamic maybe ask the user how many pages we want to fetch at the start of the script?